CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 1.10.3 (unreleased)
 ===================
 
+
+ramp_fitting
```

Comment:
super minor nit, but last time I looked at the alphabet **r**amp_fitting should come after **o**ther  ;-)